### ISSUE TITLE
Test failure with 6.9.11-55

### ISSUE THREAD

**Issue by remicollet** [Imagick/imagick#373]
Last tested version was  6.9.11-52 (test suite OK)

Using latest upstream version
```
TEST 25/280 [tests/025-get-color.phpt]
========DIFF========
001+ difference 1 not < 1 + variance 0
002+ QUANTUM g is wrong for colorString 'rgb(0, 50%, 0)': actual 32767 != expected 32768
========DONE========
FAIL Test getColor and getColorQuantum [tests/025-get-color.phpt] 
```

Looks like rounding have changed

**Comment by Danack** [comment]
This will almost certainly be something to fix upstream.

This test is there to check against places where 2^16 has been used instead of (2^16 -1), for 16bit depth ImageMagick versions.

**Comment by remicollet** [comment]
Notice: version 7 is not affected...

**Comment by remicollet** [comment]
Can be related to https://github.com/ImageMagick/ImageMagick6/commit/063e2cd36260200f34a61c2d92bbd2ac67ec5a0b

Sadly upstream bug report is not public :(

**Comment by remicollet** [comment]
Waiting for upstream on https://github.com/ImageMagick/ImageMagick6/issues/129

**Comment by remicollet** [comment]
Fixed upstream in 6.9.11-56
